demisto_sdk/commands/content_graph/interface/neo4j/queries/relationships.py

```diff
@@ -153,6 +153,7 @@ def build_depends_on_relationships_query() -> str:
 // Create the relationship, and mark as "from_metadata"
 CREATE (p1)-[r:{RelationshipType.DEPENDS_ON}{{
     mandatorily: rel_data.mandatorily,
+    target_version: rel_data.target_version,
```

Comment:
So it is part of the relationship?

Reply:
If I understand correctly your question, then yes, and now, when we add the dependencies for the metadata we'll have the version of the target pack.

Reply:
Doesn't this contradict your concern that this is not part of relationship?

Reply:
Maybe it was a misunderstood.
It has to be as part of the relationship, but It also need to be in the relationship in the graph and not only to the Pack object, this is why I added it here when we create the relationship in the graph.